topics/protocol.md

```diff
@@ -195,7 +195,7 @@ returns elements of a list.
 
 RESP Arrays are sent using the following format:
 
-* A `*` character as the first byte, followed by the number of elements in the array as a decimal number, followed by CRLF.
+* A "*" character as the first byte, followed by the number of elements in the array as a decimal number, followed by CRLF.
```

Comment:
The asterisk (`*`) has a special meaning in markdown: emphasis. It will only be converted to emphasis if it's two asterisks in a paragraph. I think it is prevented if it's in a code block (backticks). Examples:

```markdown
Hello *world

Hello *world*

Hello `*world*`
```

Result (GitHub's rendering):

> Hello *world
>
> Hello *world*
>
> Hello `*world*`

----

Backticks prevents the rendering as emphasis. (It's also possible to use backslash escapes according to the original markdown syntax: https://daringfireball.net/projects/markdown/syntax#backslash).

Reply:
So you think we should just follow line59?  with ``, the char will look like bold (emphasis)
```
use "`*`"
```

The picture below is how we currently see our website rendering

![image](https://user-images.githubusercontent.com/22811481/148928178-04573b5b-199b-47e8-8f83-3d74f0daec22.png)

Reply:
(Line 59, is the line ``For **Arrays** the first byte of the reply is "`*`"``.)

Yes, I think we can use this style also for "`+`", "`-`", "`:`", "`$`" and other places.

You don't like it?

Reply:
It is different from other chars in the original rendering

> Yes, I think we can use this style also for "+", "-", ":", "$" and other places.

but now, i can take it :)